src/OrbitGl/CallstackThreadBar.cpp

```diff
@@ -248,6 +248,10 @@ bool CallstackThreadBar::IsEmpty() const {
   return absl::StrReplaceAll(fn_name, {{"&", "&amp;"}, {"<", "&lt;"}, {">", "&gt;"}});
 }
 
+namespace {
+static std::string kErrorColorString = "#ffb000";
+}  // namespace
```

Comment:
I guess you don't need both the anonymous namespace and `static`.
You can also use `const char*` instead of `std::string` and make it `constexpr`.

Reply:
Good catch. At one point I'll remember how that all works :).